src/main/resources/templates/mdBenefitsFlow/expeditedSnapIncomeAmount.html

```diff
@@ -10,13 +10,13 @@
       <main id="content" role="main" class="form-card spacing-above-35">
         <th:block th:replace="~{fragments/icons :: moneyTips}"></th:block>
         <th:block
-            th:replace="~{fragments/cardHeader :: cardHeader(header=#{household-income-last-30-days.header}, subtext=#{household-income-last-30-days.subheader})}"/>
+            th:replace="~{fragments/cardHeader :: cardHeader(header=#{household-income-last-30-days.header}, subtext=#{household-income-last-30-days.subheader}, required=true)}"/>
         <th:block
             th:replace="~{fragments/form :: form(action=${formAction}, content=~{::formContent})}">
           <th:block th:ref="formContent">
             <div class="form-card__content">
               <th:block
-                  th:replace="~{fragments/inputs/money :: money(inputName='householdIncomeLast30Days', ariaLabel='header', placeholder='0.00')}"/>
+                  th:replace="'fragments/inputs/money' :: money(inputName='householdIncomeLast30Days', ariaLabel='header header-help-message', placeholder='0.00')"/>
             </div>
```

Comment:
Hmmm... not sure if my rebase reversed this, but we do want the `~{fragement}` syntax instead of the single quotes.

Reply:
nm, I'm taking this out as it's a duplicate field on this page now.